common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -208,6 +210,17 @@ object LogMarkerToken {
 
 }
 
+object MetricSupport {
```

Comment:
@chetanmeh wondering if we indeed need this object since we only use it for invoker. In case of controller we don't really override it in this PR.

Reply:
I thought similar support would be required for controller also so as to distinguish the metrics easily and having this change would simplify that subsequently. Its fine to drop this and have a method in `Invoker` itself for now.

Reply:
The problem with controllers would appear if we specify from the outside whether its only for activations or for crud operations. This is typically reflected in the container hostname which makes the metrics from all controller instances distinct. Given the current implementation, it would require some additional refactoring in the controller code to follow the same approach as in the invoker...

Reply:
Makes sense. Lets keep it simple then and have this method in invoker. If we rework Controller identity then this aspect can be revisited